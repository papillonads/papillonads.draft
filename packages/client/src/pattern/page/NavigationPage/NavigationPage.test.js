import React from 'react'
import navigationPageDefault from './NavigationPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<NavigationPage />', () => {
  describe('Snaphot', () => {
    test('must match navigationPageDefault', () => {
      expect(global.renderToJSON(navigationPageDefault)).toMatchSnapshot()
    })
  })
})
