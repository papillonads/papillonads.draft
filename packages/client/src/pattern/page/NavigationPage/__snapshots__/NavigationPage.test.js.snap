// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationPage /> Snaphot must match navigationPageDefault 1`] = `<div />`;
